data/assets/scene/solarsystem/heliosphere/bastille_day/actions.asset

```diff
@@ -0,0 +1,97 @@
+local bastilleday_setup_startofcme = {
+  Identifier = "os.events.bastilleday.setup.startofcme",
+  Name = "Reset to start of CME, no loop",
+  Command = [[
+    openspace.time.setTime('2000-JUL-14 10:03:00.00');
+    openspace.scriptScheduler.clear();
+  ]],
+  Documentation = "Button to reset to the the start of the CME",
+  GuiPath = "/Bastille-Day 2000",
+  IsLocal = false
+}
+
+local bastilleday_setup_shortloop = {
+  Identifier = "os.events.bastilleday.setup.shortloop",
+  Name = "Loop 10:03 - 10:16, at 2 min/second",
+  Command = [[
+    local EndtimeShortLoop = "2000 JUL 14 10:16:00";
```

Comment:
We usually do lower case for variable names and I think it would be good to do that here as well to serve as a good example for users that look at our actions to copy

Reply:
Same for a few other places further down